app/services/usps_in_person_proofing/eipp_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module UspsInPersonProofing
+  class EippHelper
+    class << self
+      def extract_vector_of_trust(sp_session)
+        sp_session['vtr']
+      end
+
+      def is_eipp?(vector_of_trust)
+        !!vector_of_trust&.first&.include?('Pe')
```

Comment:
is there a parsed `Vot::Parser::Result` we can use and grab its `enhanced_ipp?` instead?

Reply:
Ditto here. We should be passing in the `resolved_authn_context_result` and calling `resolved_authn_context_result.enhanced_ipp?` here. We should not be looking at individual vectors to determine what the authn context requirements are. That `resolved_authn_context_result` is available in the `ApplicationController`. It should be handy anywhere you would have a relevant vector of trust ([ref](https://github.com/18F/identity-idp/blob/cc149aa68805bb1e53334f4709f82195ba0d3399/app/controllers/application_controller.rb#L102)).

Reply:
Here is an example of how we do this check for biometric comparison:

https://github.com/18F/identity-idp/blob/cc149aa68805bb1e53334f4709f82195ba0d3399/app/controllers/concerns/idv_session_concern.rb#L62-L66

Reply:
See [dceae53](https://github.com/18F/identity-idp/pull/10750/commits/dceae53cdfc662f3daa64229a6cfef83ce5231f1).